frappe/client.py

```diff
@@ -312,3 +313,37 @@ def get_js(items):
 def get_time_zone():
 	'''Returns default time zone'''
 	return {"time_zone": frappe.defaults.get_defaults().get("time_zone")}
+
+@frappe.whitelist()
+def attach_file():
```

Comment:
upload_file ?

Reply:
attach_file because it only attaches file to document. No /tmp upload or non attached upload.